src/methods/createObservableMethod.js

```diff
@@ -52,5 +53,8 @@ export default function createObservableMethod (methodName, passContext) {
   }
 
   // Must be a hot stream otherwise function context may overwrite over and over again
+  if (Rx.share) {
+    return Rx.Observable.create(creator).pipe(share())
+  }
```

Comment:
rxjs.Observable

Reply:
All the `Rx` references are an object passed into the plugin, not the actual Rx package (I know that's confusing). Take a look at how `install` works:

https://github.com/vuejs/vue-rx/blob/master/src/util.js#L5